src/util/protection.cpp

```diff
@@ -4,7 +4,7 @@
p_log_ae"_J, "prop_grass_da"_J, "prop_fragtest_cnst_04"_J, "prop_thindesertfiller_aa"_J, "proc_meadowmix_01"_J, "proc_brittlebush_01"_J, "proc_stones_06"_J, "v_corp_conftable2"_J, "ng_proc_temp"_J, "v_corp_cd_wellies"_J, "v_res_rosevase"_J, "v_corp_filecabdark03"_J, "v_res_tt_porndvd01"_J, "v_res_tt_porndvd02"_J, "v_res_tt_porndvd03"_J, "v_res_tt_porndvd04"_J, "v_corp_lazychair"_J, "v_corp_go_glass2"_J, "v_serv_ct_chair01"_J, "v_corp_deskdrawfd"_J, "prop_saplin_002_b"_J, "urbangrngrass_01"_J, "v_res_fa_mag_motor"_J, "v_corp_cashpack"_J, "v_corp_lowcabdark01"_J, "prop_weed_002_ba"_J, "prop_saplin_002_c"_J, "proc_sml_reeds_01b"_J, "urbanweeds01_l1"_J, "prop_grass_dry_02"_J, "v_corp_fib_glass1"_J, "v_serv_bs_clutter"_J, "proc_sml_reeds_01c"_J, "v_res_tt_pharm3"_J, "v_res_m_h_sofa"_J, "v_serv_gt_glass1"_J, "prop_log_ad"_J, "prop_grass_dry_03"_J, "proc_grassfronds01"_J, "prop_grass_ca"_J, "v_res_tt_pharm2"_J, "prop_log_ac"_J, "v_serv_ct_monitor02"_J, "v_corp_offchairfd"_J, "proc_lizardtail_01"_J, "v_res_tt_pharm10"_J, "proc_scrub_bush01"_J, "v_corp_closed_sign"_J, "v_res_kitchnstool"_J, "v_serv_firealarm"_J, "urbandrygrass_01"_J, "v_corp_filecabdark01"_J, "proc_drygrasses01"_J, "prop_flowerweed_005_a"_J, "v_corp_lngestool"_J, "v_corp_maindeskfd"_J, "v_res_m_l_chair1"_J, "v_res_m_spanishbox"_J, "v_serv_gt_glass2"_J, "v_corp_bk_lflts"_J, "v_corp_lazychairfd"_J, "prop_tall_grass_ba"_J, "sm_hanger_portacabin"_J, "proc_desert_sage_01"_J, "v_corp_humidifier"_J, "v_corp_cashtrolley_2"_J, "v_corp_bk_lamp1"_J, "v_corp_bank_pen"_J, "prop_saplin_001_c"_J, "prop_tall_drygrass_aa"_J, "v_serv_ct_monitor01"_J, "v_corp_filecabtall_01"_J, "proc_wildquinine"_J, "v_corp_facebeanbagd"_J, "v_serv_bs_foamx3"_J, "proc_drygrasses01b"_J, "v_corp_bk_rolladex"_J, "v_corp_facebeanbagc"_J, "v_corp_banktrolley"_J, "v_corp_bk_lfltstand"_J, "proc_stones_01"_J, "v_corp_fib_glass_thin"_J, "v_serv_ct_monitor03"_J, "prop_log_ab"_J, "v_corp_facebeanbagb"_J, "v_serv_bs_cond"_J, "proc_grasses01b"_J, "v_corp_bk_lamp2"_J, "urbandryfrnds_01"_J, "v_corp_filecablow"_J, "prop_saplin_001_b"_J, "proc_lupins_01"_J, "v_serv_bs_gelx3"_J, "proc_grassdandelion01"_J, "v_corp_bk_pens"_J, "prop_log_aa"_J, "proc_indian_pbrush_01"_J, "v_corp_deskseta"_J, "v_corp_facebeanbag"_J, "proc_stones_02"_J, "proc_sage_01"_J, "v_corp_fleeca_display"_J, "proc_sml_reeds_01"_J, "proc_leafybush_01"_J, "v_serv_ct_monitor04"_J, "v_res_fa_plant01"_J, "v_serv_ct_binoculars"_J, "proc_stones_03"_J, "proc_grassplantmix_01"_J, "v_corp_deskdrawdark01"_J, "proc_meadowpoppy_01"_J, "v_serv_ct_monitor05"_J, "v_corp_filecabdark02"_J, "v_corp_desksetb"_J, "prop_grass_001_a"_J, "proc_forest_ivy_01"_J, "v_corp_lidesk01"_J, "v_corp_maindesk"_J, "proc_stones_04"_J, "v_corp_lngestoolfd"_J, "v_serv_ct_monitor06"_J, "v_serv_ct_lamp"_J, "v_corp_hicksdoor"_J, "prop_log_af"_J, "v_res_tt_cbbox"_J, "sm_hanger_mod_cabin1"_J, "sm_hanger_office_traditional"_J, "proc_stones_05"_J, "v_serv_ct_chair02"_J, "v_corp_filecabtall"_J, "v_serv_ct_monitor07"_J};
```

Comment:
This is starting to bother me, just follow this example instead of using C arrays

```cpp
constexpr auto my_cpp_fixed_size_array = std::to_array( { element_one, element_two, element_three } ):

Reply:
Try compiling it with auto. Compiler goes mad saying it cant deduce the type automatically.

Reply:
Excuse me?
![image](https://github.com/YimMenu/YimMenu/assets/837334/71d92842-64b7-4eee-adf9-f5307183d578)

Reply:
Should still work with `std::to_array` no?

Reply:
C++ 20 has a bunch of nice features, it really pays to know how to use them properly.

Reply:
I've never coded seriously in C, only time I'd ever use that is not to have to rely on the stupid Visual Studio Redistributables (for easily deployeable malware for example).

Reply:
> I've never coded seriously in C, only time I'd ever use that is not to have to rely on the stupid Visual Studio Redistributables (for easily deployeable malware for example).

:joy: C is yeah who the hell codes in original C i meant to say C++